.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+on: [push, pull_request]
+name: CI
+jobs:
+  build:
+    name: "Build on Racket '${{ matrix.racket-version }}' (${{ matrix.racket-variant }})"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        racket-version: ["7.7"]
```

Comment:
Probably good to test snapshots as well.

Reply:
I added the "current" as well as CS variants for Racket to the built, although, I am not convinced of their value to the build of plot -- "current" will be tested once the changes are merged to master anyway.  OTOH, we don't pay for the build VM time...